tests/keras/backend/backend_test.py

```diff
@@ -197,8 +197,20 @@ def test_set_learning_phase(self):
         with pytest.raises(ValueError):
             K.set_learning_phase(2)
 
-    def test_eye(self):
+    def test_variable_initializers(self):
         check_single_tensor_operation('eye', 3, WITH_NP, shape_or_val=False)
+        check_single_tensor_operation('ones', (10, 5, 1, 8), WITH_NP, shape_or_val=False)
+        check_single_tensor_operation('ones_like', (10, 5, 1, 8), WITH_NP, shape_or_val=True)
+        check_single_tensor_operation('zeros', (10, 5, 1, 8), WITH_NP, shape_or_val=False)
+        check_single_tensor_operation('zeros_like', (10, 5, 1, 8), WITH_NP, shape_or_val=True)
+
+        # random
+        check_single_tensor_operation('random_uniform_variable', (2, 3), WITH_NP,
```

Comment:
Please keep the random tests where they were. Unit tests should be small and target as few things as possible for better error reporting and debugging.